lib/vertex_client/connection.rb

```diff
@@ -42,7 +42,7 @@ def resource_config
 
     def call_with_circuit_if_available
       if VertexClient.circuit
-        VertexClient.circuit.run { yield }
+        VertexClient.circuit.run(exception: false) { yield }
```

Comment:
Double-checking: we want nil in all failure cases here so that the gem will apply a fallback rate, correct? Or could this potentially amount to silently swallowing invoice or distribute_tax requests?

Reply:
Yes, correct. You added the details around invoice and distribute below.